src/Core/Config.cs

```diff
@@ -54,6 +56,8 @@ public void SetConfig(Config config)
             Prometheus = config.Prometheus;
             CacheDal = config.CacheDal;
             CosmosDal = config.CosmosDal;
+            BurstHeader = config.BurstHeader;
+            BurstService = config.BurstService;
```

Comment:
you should set BurstService to the actual name so that it is valid. The way it is now, config.BurstService could be empty but VersionExtension.Name isn't.

Not sure I would set it here as it will create a dependency. Might make more sense to set config.BurstService = VersionExtension.Name somewhere before this.